packages/react-components/react-tree/src/utils/ImmutableSet.ts

```diff
@@ -0,0 +1,58 @@
+export class ImmutableSet<Value> {
```

Comment:
ES6 classes can have larger bundle size since it's not possible to minify as much... consider changing this to be a closure https://github.com/microsoft/fluentui/blob/master/packages/react-components/priority-overflow/src/overflowManager.ts